Library/Navigation.swift

```diff
@@ -260,6 +267,33 @@ private let routes: [String:(RouteParams) -> Decoded<Navigation>] = [
   "/users/:user_param/surveys/:survey_response_id": userSurvey
 ]
 
+private let deepLinkRoutes: [String:(RouteParams) -> Decoded<Navigation>] = allRoutes.restrict(
+  keys: [
+    "/",
+    "/mpss/:a/:b/:c/:d/:e/:f/:g",
+    "/activity",
+    "/discover",
+    "/discover/advanced",
+    "/discover/categories/:category_id",
+    "/discover/categories/:parent_category_id/:category_id",
+    "/messages/:message_thread_id",
+    "/profile/:user_param",
+    "/search",
+    "/projects/:creator_param/:project_param",
+    "/projects/:creator_param/:project_param/checkouts/:checkout_param/thanks",
+    "/projects/:creator_param/:project_param/comments",
+    "/projects/:creator_param/:project_param/creator_bio",
+    "/projects/:creator_param/:project_param/dashboard",
+    "/projects/:creator_param/:project_param/description",
+    "/projects/:creator_param/:project_param/friends",
+    "/projects/:creator_param/:project_param/posts",
+    "/projects/:creator_param/:project_param/posts/:update_param",
+    "/projects/:creator_param/:project_param/posts/:update_param/comments",
+    "/projects/:creator_param/:project_param/surveys/:survey_param",
+    "/users/:user_param/surveys/:survey_response_id"
+  ]
```

Comment:
nice! only suggestion would be to put these keys into an enum since a string could potentially change in one of these places and not the other.